lib/commands/log.js

```diff
@@ -45,6 +51,83 @@ extensions.supportedLogTypes = {
   },
 };
 
+function assignUpgradeEventListener () {
+  if (IS_UPGRADE_LISTENER_ASSIGNED && EXPRESS_SERVER.listenerCount('upgrade') !== 0) {
+    return;
+  }
+
+  // https://github.com/websockets/ws/pull/885
+  EXPRESS_SERVER.on('upgrade', (request, socket, head) => {
```

Comment:
I'm thinking about moving this part into base driver. Also it might be useful to move all websocket-related endpoints into some common root, for example `/wd/hub/ws/:sessionId:/logcat`, where `/wd/hub/ws/` will be the common prefix for all websockets. WDYT @jlipps ?

Reply:
i agree with this. if we're going to add WS stuff, let's do it in one place and provide a nice set of convenience methods for drivers to use to hook into it.